GUIDE.md

```diff
@@ -0,0 +1,167 @@
+
+1. The next step uses [Juan Parrilla's git repository](https://github.com/jparrill/racadm-image) for simplicity to do the following:
+
+   - Create a `podman` container that will setup `iDRAC` on DELL servers to boot from ISO
+
+   > NOTE: Steps 14-18 only work on DELL hardware. A different method would need to be used if using a different vendor to mount your live ISO.
+
+1)  Create a webserver container labeled mywebserver that is to serve the `live.iso` from the `~/assisted-installer/images` directory serving out of port 8080 as follows
+
+        [$USER@assisted_installer ~]# firewall-cmd --add-port=8080/tcp --zone=public --permanent
+        [$USER@assisted_installer ~]# firewall-cmd --reload
+        [$USER@assisted_installer ~]# podman run -d --name mywebserver -v ~assisted-installer/images/:/var/www/html:Z -p 8080:8080/tcp registry.centos.org/centos/httpd-24-centos7:latest
+
+    > NOTE: Verify you can access the live.iso link as such http://<host-ip>:8080/live.iso
+
+1. Clone the repository as follows
+
+
+        ~~~sh
+        [$USER@assisted_installer ~]# cd ~/assisted-installer
+
+        [$USER@assisted_installer assisted-installer]# git clone https://github.com/jparrill/racadm-image.git
+        ~~~
+
+1.  Change into the `racadm-image` directory and build the Dockerfile using `podman`. This will create an podman container that will be used to mount the live.iso to your baremetal nodes via iDRAC.
+
+        [$USER@assisted_installer assisted-installer]# cd racadm-image
+        [$USER@assisted_installer racadm-image]# podman build . -t idracbootfromiso
+
+    > NOTE: Ignore any errors and just ensure your `idracbootfromiso` image is created and exists under the command podman images. If it failed, please attempt to re-run the command above.
+
+1.  With the `idracbootfromiso` image created, we will now use it to mount the `live.iso` on all of our OpenShift nodes. For simplicity, this example shows 3 servers (3 masters, 0 workers) but can be extended to X number of servers requiring the `live.iso`
+
+        [$USER@assisted_installer ~]# for i in <master0-idrac-ip> <master1-idrac-ip> <master2-idrac-ip>; do podman run --net=host idracbootfromiso -r $i -u <idrac-user> -p "<idrac-pw>" -i http://       <host-ip>:8080/live.iso; done
+
+    > NOTE1: Ensure to include the proper host **iDRAC IPs**, **iDRAC user** and **iDRAC password**. This for loop assumes iDRAC user and iDRAC password are the same, if different adjust the shell command accordingly.
+
+
+    > NOTE2: Also make sure to **remove/eject the older ISO** which were mounted using `racadm` command as shown below
+
+        sshpass -p '*****' ssh root@<iDRAC-IP> racadm remoteimage -d
+
+1.  The `live.iso` should reboot the nodes and boot them into a Fedora Live image. Once it has done this, shortly you will notice the nodes becoming discoverable for your cluster via the `http://<host-ip>:6008/clusters/<cluster-id>` dashboard.
```

Comment:
If we get the Ansible playbook merged, this will all change.

Reply:
it can be amended later :)

Reply:
yeah, also we can keep this guide and provide steps for ansible in a sub-section? just a suggestion.